### PR TITLE
buildPythonPackage: fix typos

### PR DESCRIPTION
**Pull request by kragniz**

### CODE REVIEW

**Comment by domenkozar** [comment]
Hmm, this will rebuild a lot of python packages. Best if we merge it together with some other work.

**Comment by domenkozar** [comment]
Thank btw! :)

File: pkgs/development/python-modules/generic/default.nix [hunk 1]
```diff
@@ -112,7 +112,7 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs ["disabled" "doCheck"] //
   postFixup = attrs.postFixup or ''
     wrapPythonPrograms
 
-    # check if we have two packagegs with the same name in closure and fail
+    # check if we have two packages with the same name in closure and fail
     # this shouldn't happen, something went wrong with dependencies specs
```
Comment (FRidh) [inline pkgs/development/python-modules/generic/default.nix]: if we move comments out of the arguments, then future changes should not cause any rebuilds, right?

Comment (domenkozar) [inline pkgs/development/python-modules/generic/default.nix]: indeed

**Comment by domenkozar** [comment]
@FRidh could we also take these changes to `python-wip`?

**Comment by domenkozar** [comment]
Fixed in 9acba92687520d715d994509f6192704b3cc48fb, will land shortly in master

**Pull request closed without merging**